.github/workflows/private-pki-test.yaml

```
@@ -14,6 +14,7 @@ on:
     paths:
       - ".github/workflows/private-pki-test.yaml"
       - "bundles/k3d-standard/**"
+      - "test/private-pki/**"
```

```suggestion
      - "test/playwright/private-pki/**"
```

If I'm not mistaken?

---

fixed. good catch